### PR TITLE
fbp-to-dot: Abort if sol_init fails v2

### PR DESCRIPTION
**Pull request by anselmolsm**
Also change the return value when errors occur.

Signed-off-by: Anselmo L. S. Melo <anselmo.melo@intel.com>

### CODE REVIEW

File: src/bin/sol-fbp-to-dot/main.c [hunk 1]
```diff
@@ -337,7 +340,8 @@ main(int argc, char *argv[])
 
     if (convert_fbp_to_dot(&input_graph, dotfile)) {
```
Comment (ricardotk) [inline src/bin/sol-fbp-to-dot/main.c]: I would remove the brackets here and change the if condition to

if (convert_fbp_to_dot(&input_graph, dotfile) != 0)
   r = EXIT_SUCCESS;
else
    fprintf(stderr, "Couldn't convert from fbp to ...

**Comment by ricardotk** [comment]
LGTM with comments,  if nobody else spots anything, go ahead and merge it.

File: src/bin/sol-fbp-to-dot/main.c (hunk 1 above)
Comment (anselmolsm) [inline src/bin/sol-fbp-to-dot/main.c]: OK. But in fact, convert_fbp_to_dot returns 0 on success ;)

**Comment by anselmolsm** [comment]
Applied, tks

**Pull request closed without merging**